### ISSUE TITLE
We should verify ManufacturersModelName, DeviceSerialNumber, and SoftwareVersion on the ET computer during the next scanning session

### ISSUE THREAD

**Issue by esavary** [TheAxonLab/hcph-sops#238]

**Comment by acionca** [comment]
@esavary let's not forget to do it this next session

**Comment by oesteban** [comment]
Except for the software version, I think you want to do that for both the infrared lamp and the camera.

**Comment by acionca** [comment]
For the ET camera:
![Uploading IMG20231023190517.jpg …]()

**Comment by oesteban** [comment]
@acionca the link above is self-recursive...

**Comment by acionca** [comment]
> @acionca the link above is self-recursive...

Inded, the phone upload didn't work I guess. Here are all the details from the model:
>EyeLink CL-OC
Serial # ZAB49
Power: 3.6 - 5.7V @ 700mA
Class 1 laser device
IEC 60825-1 Ed.1.2:2001

**Comment by oesteban** [comment]
We should write down these details here https://github.com/TheAxonLab/hcph-sops/blob/2e873b737b46c46f80d9906b576d401d9288012d/study-settings.yml#L25-L30

**Comment by oesteban** [comment]
Any updates on this? @acionca @esavary

**Comment by esavary** [comment]
I have included the serial number and model name in the JSON files for the ET data, but I haven't added them to the SOPs yet. It seems that the references for the respiration belt and ECG are also missing. I will try to find the information for the belt and ECG tonight and add everything at once.

**Comment by oesteban** [comment]
I think the other physio recordings should be okay after merging #318 (see cell 4 of https://output.circle-artifacts.com/output/job/286f0227-748d-47af-ba4f-91a4b8ecb3ec/artifacts/0/_SOPs_preview/data-management/physio-to-bids/index.html).

Let's focus this one on the ET and if there's something about the other physio to be addressed, let's do that in #318 or separate issues/PRs.

**Comment by esavary** [comment]
Yes, I have uploaded the files on Dropbox, so I'm closing the issue. @celprov feel free to reopen if necessary.

**Comment by esavary** [comment]
Sorry, I wanted to close another issue.
